frontend/src/components/requests/index.jsx

```diff
@@ -41,10 +40,6 @@ class Requests extends React.Component {
         this.triggerSubmit = this.triggerSubmit.bind(this);
         this.confirmEventSubmission = this.confirmEventSubmission.bind(this);
         this.cancelEventSubmission = this.cancelEventSubmission.bind(this);
-
-        this.state = {
```

Comment:
Using the Redux state here rather than duplicating it is what fixed the bug where filters didn't stay applied after you triggered an event